pom.xml

```diff
@@ -764,6 +765,7 @@
                         <version>${munit.extensions.maven.plugin.version}</version>
                         <configuration>
                             <munitFailIfNoTests>false</munitFailIfNoTests>
+                            <minMuleVersionConfig>CONNECTOR</minMuleVersionConfig>
```

Comment:
Just to know, what does it mean? That [class](https://github.com/mulesoft/munit-runtime/blob/ad0c17ce0b84e0c318e61bfc9a350e5ddc88ae85/munit-remote/src/main/java/org/mule/munit/remote/MinMuleVersionConfig.java#L16) lacks javadoc.

Reply:
This parameter indicates which runtime version you want to run Munit tests with.
If CONNECTOR is selected, the version declared in minMuleVersion in the connector's POM will be used.
If RUNTIME is set, the minMuleVersion value passed as the mvn parameter will be used.
By default, both versions are run.